models/core-java/src/main/java/org/commonjava/indy/model/core/RemoteRepository.java

```diff
@@ -38,6 +38,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger( RemoteRepository.class );
 
+    public static final String PREFETCH_LISTING_TYPE_KOJI = "koji";
```

Comment:
Can we put this constant in the koji add-on codebase?

Reply:
so maybe the KojiListContentBuilder should also be moved to.

Reply:
Yeah, if possible. If it creates a mess in terms of module inter-dependency, then I guess we can fudge it using koji mentions in the subsystem...but I'd rather keep it self-contained in the koji add-on of we can.